Panic if unable to create the log file

```./goval-dictionary  fetch-debian 9
panic: open /var/log/vuls/goval-dictionary.log: permission denied

goroutine 1 [running]:
github.com/inconshreveable/log15.must(...)
        /home/ubuntu/go/pkg/mod/github.com/inconshreveable/log15@v0.0.0-20180818164646-67afb5ed74ec/handler.go:340
github.com/inconshreveable/log15.muster.FileHandler(0xc000091680, 0x22, 0xc0cde0, 0xb5ecf0, 0x22, 0x0)
        /home/ubuntu/go/pkg/mod/github.com/inconshreveable/log15@v0.0.0-20180818164646-67afb5ed74ec/handler.go:348 +0x8a
github.com/kotakanbe/goval-dictionary/util.SetLogger(0xb3649e, 0xd, 0xc000000000)
        /home/ubuntu/go/src/github.com/kotakanbe/goval-dictionary/util/util.go:62 +0x227
github.com/kotakanbe/goval-dictionary/commands.(*FetchDebianCmd).Execute(0xc000257450, 0xc184e0, 0xc000092000, 0xc00027c540, 0x0, 0x0, 0x0, 0x0)
        /home/ubuntu/go/src/github.com/kotakanbe/goval-dictionary/commands/fetch-debian.go:97 +0x10a
github.com/google/subcommands.(*Commander).Execute(0xc0000a8000, 0xc184e0, 0xc000092000, 0x0, 0x0, 0x0, 0xc0002c6580)
        /home/ubuntu/go/pkg/mod/github.com/google/subcommands@v1.0.1/subcommands.go:142 +0x2f9
github.com/google/subcommands.Execute(...)
        /home/ubuntu/go/pkg/mod/github.com/google/subcommands@v1.0.1/subcommands.go:420
main.main()
        /home/ubuntu/go/src/github.com/kotakanbe/goval-dictionary/main.go:52 +0x463```

```ubuntu@dev  ~│g│s│g│k│goval-dictionary  ⎇ master~  ./goval-dictionary fetch-ubuntu 18
panic: open /var/log/vuls/goval-dictionary.log: permission denied

goroutine 1 [running]:
github.com/inconshreveable/log15.must(...)
        /home/ubuntu/go/pkg/mod/github.com/inconshreveable/log15@v0.0.0-20180818164646-67afb5ed74ec/handler.go:340
github.com/inconshreveable/log15.muster.FileHandler(0xc000119680, 0x22, 0xbe6840, 0xb34270, 0x22, 0x0)
        /home/ubuntu/go/pkg/mod/github.com/inconshreveable/log15@v0.0.0-20180818164646-67afb5ed74ec/handler.go:348 +0x8a
github.com/kotakanbe/goval-dictionary/util.SetLogger(0xb0b14d, 0xd, 0xc000000000)
        /home/ubuntu/go/src/github.com/kotakanbe/goval-dictionary/util/util.go:62 +0x227
github.com/kotakanbe/goval-dictionary/commands.(*FetchUbuntuCmd).Execute(0xc0002df4f0, 0xbf1840, 0xc00011a000, 0xc000304540, 0x0, 0x0, 0x0, 0x12387e0)
        /home/ubuntu/go/src/github.com/kotakanbe/goval-dictionary/commands/fetch-ubuntu.go:96 +0xfe
github.com/google/subcommands.(*Commander).Execute(0xc000130000, 0xbf1840, 0xc00011a000, 0x0, 0x0, 0x0, 0xc00034e580)
        /home/ubuntu/go/pkg/mod/github.com/google/subcommands@v1.0.1/subcommands.go:142 +0x2f9
github.com/google/subcommands.Execute(...)
        /home/ubuntu/go/pkg/mod/github.com/google/subcommands@v1.0.1/subcommands.go:420
main.main()
        /home/ubuntu/go/src/github.com/kotakanbe/goval-dictionary/main.go:55 +0x461```

--- a/util/util.go
+++ b/util/util.go
@@ -58,10 +58,15 @@ func SetLogger(logDir string, quiet, debug, logJSON bool) {
 	var hundler log15.Handler
 	if _, err := os.Stat(logDir); err == nil {
 		logPath := filepath.Join(logDir, "goval-dictionary.log")
-		hundler = log15.MultiHandler(
-			log15.Must.FileHandler(logPath, logFormat),
-			lvlHundler,
-		)
+		if err := ioutil.WriteFile(logPath, []byte{}, 0700); err != nil {
+			log15.Error("Failed to create a log file", "err", err)
+			hundler = lvlHundler
+		} else {
+			hundler = log15.MultiHandler(
+				log15.Must.FileHandler(logPath, logFormat),
+				lvlHundler,
+			)
+		}
 	} else {
 		hundler = lvlHundler
 	}

